types/params_test.go

```diff
@@ -451,6 +453,26 @@ func TestConsensusParamsUpdate(t *testing.T) {
 			},
 			updatedParams: makeParams(makeParamsArgs{blockBytes: 1, blockGas: 2, evidenceAge: 3, pbtsHeight: 1}),
 		},
+		// update enabled sbt height only
+		{
+			intialParams: makeParams(makeParamsArgs{blockBytes: 1, blockGas: 2, evidenceAge: 3, sbtEnableHeight: 1}),
```

Comment:
```suggestion
			intialParams: makeParams(makeParamsArgs{blockBytes: 1, blockGas: 2, evidenceAge: 3, sbtEnableHeight: 0}),
```

Should we set sbtEnableHeight to 0 or remove it? Otherwise, the 2 test cases are the same.